### PR TITLE
Two dimensional matrix

### PR DESCRIPTION
**Pull request by YESENIA2012**
The first algorithm enters elements into the array and the second algorithm is to loop through the elements of an array

### CODE REVIEW

**Comment by YESENIA2012** [comment]
can you review it? please.

File: introducingElementsTwoDimensionalArray.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      );
+      contador2 = contador2 + 1;
+    }
+  }
+  return matrixCopia;
+}
+
+var introducir = introducirElementosMatrix(matrix).then(function (matrixCopia) {
+  console.log("matriz copia: ", matrixCopia);
+});
```
Comment (erick2014) [inline introducingElementsTwoDimensionalArray.js]: remove this code from line 29 to 31

File: introducingElementsTwoDimensionalArray.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const lea = require("prompt-async");
+
+var matrix = [
+  [, ,],
+  [, ,],
+  [, ,],
+];
```
Comment (erick2014) [inline introducingElementsTwoDimensionalArray.js]: remove this variable

File: loopThroughElementsArray.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+var vector = [
+  [1, 2],
+  [3, 4],
+  [5, 6],
+];
+
+for (contador = 0; contador < vector.length; contador++) {
+  let filaMatrix = vector[contador];
+  console.log("Columnas en la fila:", contador);
```
Comment (erick2014) [inline loopThroughElementsArray.js]: translate the message of the console.log to english

File: loopThroughElementsArray.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+var vector = [
+  [1, 2],
+  [3, 4],
+  [5, 6],
+];
+
+for (contador = 0; contador < vector.length; contador++) {
```
Comment (erick2014) [inline loopThroughElementsArray.js]: translate variable name to english : counter

File: loopThroughElementsArray.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  let filaMatrix = vector[contador];
```
Comment (erick2014) [inline loopThroughElementsArray.js]: the variable name should be in english: `matrixRow`

File: loopThroughElementsArray.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  console.log("Columnas en la fila:", contador);
+
+  for (contador2 = 0; contador2 < filaMatrix.length; contador2++) {
```
Comment (erick2014) [inline loopThroughElementsArray.js]: translate variable name to english : counter2

File: loopThroughElementsArray.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    var columnaMatrix = filaMatrix[contador2];
```
Comment (erick2014) [inline loopThroughElementsArray.js]: translate variable name to english : `matrixColumn`

File: test/introducingElements.test.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    expect(resultado[0]).to.have.lengthOf(3);
+    expect(randomNumbers[1]).to.eql(resultado[0][1]);
+    expect(resultado[1]).to.have.lengthOf(3);
+  });
+
+  it("should insert random numbers into a 3 * 3 matrix", async function () {
+    const matrix = [
+      [, ,],
+      [, ,],
+      [, ,],
```
Comment (erick2014) [inline test/introducingElements.test.js:47]: remove commas inside each vector: 
```
const matrix = [
      [ ],
      [ ],
      [ ],
]
```

File: test/introducingElements.test.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  });
+  beforeEach(function () {
+    randomNumbers = [];
+  });
+
+  it("should insert random numbers into a 2 * 3 matrix", async function () {
+    const matrix = [
+      [, , ,],
+      [, , ,],
+    ];
```
Comment (erick2014) [inline test/introducingElements.test.js:32]: remove commas inside each vector: 
```
const matrix = [
      [ ],
      [ ]
]
```

File: test/introducingElements.test.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
+var expect = require("chai").expect;
+const sinon = require("sinon");
+const lea = require("prompt-async");
+const mainfunction = require("../introducingElementsTwoDimensionalArray");
+// const functionUnderTest = require("../introducingElementUnderTest");
```
Comment (erick2014) [inline test/introducingElements.test.js]: remove this comment

**Review by erick2014 — CHANGES_REQUESTED** [review]

**Comment by YESENIA2012** [comment]
changes were made

**Pull request merged**